uspace/app/download/main.c

```diff
@@ -242,7 +242,7 @@ int main(int argc, char *argv[])
 		return EIO;
 	}
 
-	return EOK;
+	return 0;
```

Comment:
This is travelling in the opposite direction.

Reply:
Huh. Weird.

Reply:
Ah, right, that's `main()`. Literal 0 is correct there, unless we make a convention of returning `EXXX` from `main()`.

Reply:
The story here is that I wrapped all errno returns from main in `EXIT_RC()` to get it to typecheck (simultaneously changing EOKs to 0 there), then I removed the macro in one of the last commits.